dashboards/docs/rss_reader.md

```diff
@@ -0,0 +1,9 @@
+This report answers the following questions:
+
+What is the channel information for each feed?
+How many items are there based on each category?
+How many items are there based on each author?
+How many items are there based on each month?
+What is the author’s name for each item?
+When was items published?
+What are the categories of each item?
```

Comment:
```suggestion
- What is the channel information for each feed?
- How many items were published for each category?
- How many items were published by each author?
- How many items were published by month or year?
- What items were published?
```

Please remember to include proper formatting (missing the bullet points significantly changes the doc format).